plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -45,28 +49,29 @@ def __init__(
         hazard: str,
         folder_name: str,
         together_api_key: TogetherApiKey,
-        benign_prompts: bool = False,
+        persona_type: str = "typical",
```

Comment:
Given more options, I think we should remove the default here and make everything explicit.